src/views/Dissolution/Affidavit.vue

```diff
@@ -18,7 +18,7 @@ import { ActionBindingIF, UploadAffidavitIF } from '@/interfaces'
     Affidavit
   }
 })
-export default class UploadAffidavit extends Mixins(CommonMixin) {
```

Comment:
The classname is thrown away by the index.ts file (which imports default as XXX, thus assigning the class name at that time).